src/test/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/data/CertificatesTest.java

```diff
@@ -237,11 +237,6 @@ void negativePopulateCertificateExpiresWithin30Days() throws Exception {
         keyStoreMockedStatic.verify(() -> KeyStore.getInstance(config.getPncCertificateLocation(),
```

Comment:
Cert test can be deleted as we can no longer check certificates

Reply:
The expiry job is still needed for the dashboard stats. so the idea would be to copy the cert from the PNC server into the dashboard to get the stat using the job.